rust/lance-index/src/vector/graph/memory.rs

```diff
@@ -22,13 +22,16 @@ use lance_linalg::{distance::MetricType, MatrixView};
 
 /// All data are stored in memory
 pub struct InMemoryVectorStorage {
```

Comment:
This gets quite large right?  What happens when it is larger than the RAM on the system?  Does it panic or fail with an error?  Or is this constrained somehow?

Reply:
it's used for tests only